php-fpm.dockerfile

```diff
@@ -5,8 +5,12 @@ RUN apt-get update -y && apt-get install -y nginx
 WORKDIR /app/laravel-example
 
 COPY . /app
-RUN chown -R www-data:www-data /app
+RUN chown -R www-data:www-data /app/laravel-example/storage /app/laravel-example/database && \
+    chmod +x /app/fpm-entrypoints/*
+RUN yes| php artisan migrate
```

Comment:
Does it make sense to commit the sqlite db file tho and remove this line?

Reply:
it does not because you have migrations in place already